src/cloud-api-adaptor/test/e2e/common.go

```diff
@@ -23,6 +23,7 @@ import (
 )
 
 const BUSYBOX_IMAGE = "quay.io/prometheus/busybox:latest"
+const BUSYBOX_CURL_IMAGE = "quay.io/curl/curl:latest"
```

Comment:
nit: i don't think that's busybox, that's alpine probably.

Reply:
Yes thats an alpine image, Renaming things.